barretenberg/cpp/src/barretenberg/vm2/constraining/relations/ecc.test.cpp

```diff
@@ -460,5 +476,211 @@ TEST(EccAddConstrainingTest, EccNegativeBadDouble)
     EXPECT_THROW_WITH_MESSAGE(check_relation<ecc>(trace, ecc::SR_OUTPUT_X_COORD), "OUTPUT_X_COORD");
 }
 
+TEST(ScalarMulConstrainingTest, ScalarMulEmptyRow)
+{
+    auto trace = TestTraceContainer::from_rows({
+        { .precomputed_clk = 1 },
+    });
+
+    check_relation<scalar_mul>(trace);
+}
+
+TEST(ScalarMulConstrainingTest, MulByOne)
+{
+    NoopEventEmitter<EccAddEvent> ecc_add_event_emitter;
+    EventEmitter<ScalarMulEvent> scalar_mul_event_emitter;
+    EccSimulator ecc_simulator(ecc_add_event_emitter, scalar_mul_event_emitter);
+
+    FF scalar = FF(1);
+    ecc_simulator.scalar_mul(p, scalar);
+
+    TestTraceContainer trace = TestTraceContainer::from_rows({
+        { .precomputed_first_row = 1 },
```

Comment:
It seems like you are using this row if you have it here :) so it makes even more sense to add it in the empty test (or remove it from here if it's not needed).